src/devices/Vl53L1X/Vl53L1X.cs

```diff
@@ -0,0 +1,959 @@
+            0x80, /* 0x67 : Min count Rate LSB */ 0x08, /* 0x68 : not user-modifiable */
+            0xb8, /* 0x69 : not user-modifiable */ 0x00, /* 0x6a : not user-modifiable */
+            0x00, /* 0x6b : not user-modifiable */
+            0x00, /* 0x6c : Intermeasurement period MSB, 32 bits register, use SetIntermeasurementInMs() */
+            0x00, /* 0x6d : Intermeasurement period */ 0x0f, /* 0x6e : Intermeasurement period */
+            0x89, /* 0x6f : Intermeasurement period LSB */ 0x00, /* 0x70 : not user-modifiable */
+            0x00, /* 0x71 : not user-modifiable */
+            0x00, /* 0x72 : distance threshold high MSB (in mm, MSB+LSB), use SetD:tanceThreshold() */
+            0x00, /* 0x73 : distance threshold high LSB */
+            0x00, /* 0x74 : distance threshold low MSB ( in mm, MSB+LSB), use SetD:tanceThreshold() */
+            0x00, /* 0x75 : distance threshold low LSB */ 0x00, /* 0x76 : not user-modifiable */
+            0x01, /* 0x77 : not user-modifiable */ 0x0f, /* 0x78 : not user-modifiable */
+            0x0d, /* 0x79 : not user-modifiable */ 0x0e, /* 0x7a : not user-modifiable */
+            0x0e, /* 0x7b : not user-modifiable */ 0x00, /* 0x7c : not user-modifiable */
+            0x00, /* 0x7d : not user-modifiable */ 0x02, /* 0x7e : not user-modifiable */
+            0xc7, /* 0x7f : ROI center, use SetROI() */ 0xff, /* 0x80 : XY ROI (X=Width, Y=Height), use SetROI() */
+            0x9B, /* 0x81 : not user-modifiable */ 0x00, /* 0x82 : not user-modifiable */
+            0x00, /* 0x83 : not user-modifiable */ 0x00, /* 0x84 : not user-modifiable */
+            0x01, /* 0x85 : not user-modifiable */ 0x00, /* 0x86 : clear interrupt, use ClearInterrupt() */
+            0x00 /* 0x87 : start ranging, use StartRanging() or StopRanging(), If you want an automatic start after VL53L1X_init() call, put 0x40 in location 0x87 */
+        };
+
+        private bool _rangingInitialized;
+
+        /// <summary>
+        /// Creates a Vl53L1X sensor class
+        /// </summary>
+        /// <param name="i2CDevice">The I2C Device</param>
+        /// <param name="operationTimeoutMilliseconds">Timeout for reading data, by default 500 milliseonds</param>
+        /// <param name="shouldDispose">True to dispose the I2C Device at dispose</param>
+        public Vl53L1X(I2cDevice i2CDevice, int operationTimeoutMilliseconds = 500, bool shouldDispose = true)
+        {
+            _operationTimeoutMilliseconds = operationTimeoutMilliseconds;
+            _shouldDispose = shouldDispose;
+            _i2CDevice = i2CDevice;
+            WaitForBooted();
+            InitSensor();
+        }
+
+        /// <summary>
+        /// Get a distance in millimeters.
+        /// If ranging has not been started yet, the function will automatically start the ranging feature of the device.
+        /// </summary>
+        public ushort Distance
```

Comment:
This one is actually better as a function GetDistance. Reason is because a property should be almost not changing (I know there are exceptions like DateTime.Now and few others). 
We do still have qite a few with properties anyway. So maybe add a function as well if you want to keep the property (again, we have both)

Reply:
And as it's a distance, we should use UnitsNet for this.

Reply:
I added an additional function GetDistance() like the sister device Vl53L0X and switched from ushort to Length